src/main/kotlin/apply/application/RecruitmentDtos.kt

```diff
@@ -42,3 +42,52 @@ data class RecruitmentItemRequest(
     @field:NotBlank
     var description: String = ""
 )
+
+data class ApplicationFormSaveRequest(
+    @field:Size(min = 0, max = 255)
+    val referenceUrl: String,
+
+    @field:NotNull
+    val isSubmit: Boolean,
```

Comment:
`isSubmitted` 또는 `submitted`가 맞습니다.

```suggestion
    val isSubmitted: Boolean,
```

Reply:
수정했습니다...!